integrations/SD_Lon/sd_importer.py

```diff
@@ -7,10 +7,11 @@
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 #
 import os
-import pickle
-import hashlib
+import sys
 import datetime
 from anytree import Node
+
+sys.path.append('../')
 import ad_reader
```

Comment:
The need for adding a path in order to complete the imports may point towards an underlying issue regarding the structure of the codebase that we should address rather sooner than later.

Reply:
agree

Reply:
Det er ikke noget jeg kan flytte på kort sigt, det kræver omlægning til et modul, jeg vil gerne have lov at beholde denne syntax indtil da.